workspaces.tf

```diff
@@ -12,6 +12,7 @@ locals {
       /* Default settings for all fleets/workspaces. */
 
       boot_hosts_count = 2
+      store_hosts_count = 2
```

Comment:
Honestly, now that I think about it, do we really need two nodes per DC for the test fleet?

Reply:
Maybe to test write operations to the DB?

Reply:
Fair enough. But then we don't need two boot nodes do we?

Reply:
https://github.com/status-im/infra-shards/pull/11